sql/2021/ecommerce/median_lighthouse_score_ecommsites.sql

```diff
@@ -0,0 +1,25 @@
+#standardSQL
+# 13_20: Lighthouse category scores per eCommerce plaforms. Web Almanac run LightHouse only in mobile mode and hence references to mobile tables
+# See https://github.com/HTTPArchive/almanac.httparchive.org/pull/1087#issuecomment-684983795. Due to this, using Sep-2020 in this query
+SELECT
+  app AS ecommVendor,
+  COUNT(DISTINCT url) AS freq,
+  APPROX_QUANTILES(CAST(JSON_EXTRACT_SCALAR(report, '$.categories.performance.score') AS NUMERIC), 1000)[OFFSET(500)] AS median_performance,
+  APPROX_QUANTILES(CAST(JSON_EXTRACT_SCALAR(report, '$.categories.accessibility.score') AS NUMERIC), 1000)[OFFSET(500)] AS median_accessibility,
+  APPROX_QUANTILES(CAST(JSON_EXTRACT_SCALAR(report, '$.categories.pwa.score') AS NUMERIC), 1000)[OFFSET(500)] AS median_pwa,
+  APPROX_QUANTILES(CAST(JSON_EXTRACT_SCALAR(report, '$.categories.seo.score') AS NUMERIC), 1000)[OFFSET(500)] AS median_seo,
+  APPROX_QUANTILES(CAST(JSON_EXTRACT_SCALAR(report, '$.categories.best-practices.score') AS NUMERIC), 1000)[OFFSET(500)] AS median_best_practices
+FROM
+  `httparchive.sample_data.lighthouse_mobile_*`
```

Comment:
Please go through all queries to ensure that there are no `sample_data` tables in use. Those subsets are more for iterating on queries and not intended to be used in the reader-facing stats.
```suggestion
  `httparchive.lighthouse.2021_07_01_mobile`
```

Reply:
Thanks. I will fix this query. I noticed I had forgotten to run this so I have run it with the `2021_07_01` actual set